packages/web-components/src/accordion/accordion.ts

```diff
@@ -79,28 +79,19 @@ export class Accordion extends FASTElement {
     }
   }
 
-  private activeItemIndex: number = 0;
-
   /**
-   * Find the first expanded item in the accordion
-   * @returns {void}
+   * Find the first expanded item in the accordion.
+   *
+   * @internal
    */
-  private findExpandedItem(): BaseAccordionItem | Element | null {
-    if (this.accordionItems.length === 0) {
-      return null;
-    }
-
-    return (
-      this.accordionItems.find(
-        (item: Element | BaseAccordionItem) => item instanceof BaseAccordionItem && item.expanded,
-      ) ?? this.accordionItems[0]
-    );
+  private findExpandedItem(): BaseAccordionItem | null {
```

Comment:
I think based on the changes this can't return `null` anymore, only `undefined`?